### DISCUSSION TITLE
How to make Dolphin full glass?

### DISCUSSION THREAD

**Discussion by aliquo93** [Luwx/Lightly#152] [Q&A]
How to make Dolphin full glass like this? 

![Lightly-fullglass](https://user-images.githubusercontent.com/9071297/150661778-082f2707-22fa-4cf5-9719-85d5f4dd830e.png)

**Comment by jonhteper** [comment]
Is not exactly equal, but you can based in [this themes](https://github.com/jonhteper/lightly-color-schemes)

**Reply by aliquo93** [reply]
Thanks, I've set it!
